embedded-graphics/src/primitives/mod.rs

```diff
@@ -308,6 +308,51 @@ macro_rules! egline {
     }};
 }
 
+/// Create a [`Polyline`](./primitives/polyline/struct.Polyline.html) with optional styling using a
+/// convenient macro.
```

Comment:
I know that this is the same as the other macros, but I think we should change this. The macros never create a raw primitive and always apply styling.

Now that I'm thinking about this, does having the macro variant without `style = ...` make sense at all? Why would you want to create a styled primitive that doesn't draw anything? Maybe it would be more useful to return a raw primitive in that case.

Reply:
Yeah, the macro docs need an overhaul. There's lots of semi-incorrect copypasta. 

> Now that I'm thinking about this, does having the macro variant without style = ... make sense at all?

Not in their current form. We could change the variants without `style =` return a plain primitive, but that might get confusing. We could also remove the macros completely (although worth consulting with the e-g matrix chat if we do this).

Do you mind if we move this specific change and the other macro stuff to another PR? I'll open an issue to track.

Reply:
See #350

Reply:
> We could also remove the macros completely (although worth consulting with the e-g matrix chat if we do this).

TBH, I never use the macros and don't see a huge benefit in having them, so I would support this idea.

Reply:
> Do you mind if we move this specific change and the other macro stuff to another PR? I'll open an issue to track.

No, I just noticed this while doing the review and it should be handled in another PR.